File: Level 1/둘만의 암호/solution.js
function solution(s, skip, index) {
  const alphabets = [..."abcdefghijklmnopqrstuvwxyz"].filter(
    (alphabet) => !skip.includes(alphabet)
  );

  return [...s]
    .map(
      (char) => alphabets[(alphabets.indexOf(char) + index) % alphabets.length]
    )
    .join("");
}

File: Level 1/둘만의 암호/solution.py
def solution(s, skip, index):
    alphabets = [
        alphabet for alphabet in "abcdefghijklmnopqrstuvwxyz" if alphabet not in skip
    ]

    return "".join(
        alphabets[(alphabets.index(char) + index) % len(alphabets)] for char in s
    )
